spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -849,8 +847,7 @@ def add_all_to_nwbfile(recording: se.RecordingExtractor, nwbfile=None,
             nwbfile=nwbfile,
             use_timestamps=use_timestamps,
             metadata=metadata,
-            write_as_lfp=write_as_lfp,
-            return_scaled=return_scaled
```

Comment:
Why not leaving the option to use `return_scaled=True`? I agree it should be `False` by default though

Reply:
Because the trace access here is fundamentally intended to be False, not just by default. Nwb series objects always behave like the equivalent of `get_traces(..., return_scaled=False)` with `gains` set. The only issue we would encounter here is if a given extractor doesn't support `return_scaled=False` (say, it only records them as pre-converted floats), but if no extractor comes immediately to mind on that we can always wait until the situation arises. Even in that case, you'd simply write the series with `return_scaled=True` and conversion factor of `1e-6`.